catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java

```diff
@@ -15,7 +15,7 @@
 
 import static org.apache.commons.lang.Validate.notNull;
 
-import java.awt.Graphics2D;
+import java.awt.*;
```

Comment:
This package should be "java.awt.Graphics2D" instead as check-style will only allow importing specific classes.